OpenSim/Simulation/Model/ModelVisualizer.cpp

```diff
@@ -245,7 +245,7 @@ bool ModelVisualizer::
 findGeometryFile(const Model& aModel, 
                  const std::string&          geoFile,
                  bool&                       geoFileIsAbsolute,
-                 SimTK::Array_<std::string>& attempts) const
+                 SimTK::Array_<std::string>& attempts)
```

Comment:
Thanks @chrisdembia Yes, now a static method so can't be const since there's no object under it.